### PR TITLE
add popup on button

### PR DESCRIPTION
**Pull request by alphym**
<h3>add popup on see project button</h3>

- apply popup on see  project button on both desktop and mobile version 

image

- due to my partners' absence, I was allowed to work alone this week.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -472,6 +472,61 @@ <h3>Contact me</h3>
...
+                            <li><a href="#">javascript</a></li>
+                        </ul>
+                        <div class="modal-btn-desk-view">
+                            <a class="modal-desk-btn">See live <img src="./images/modal-live.png" alt="" style="height: 18px; width: 18px;"> </a>
+                            <a class="modal-desk-btn"> See source<img src="./images/modal-source.png" alt="" style="height: 18px; width: 18px;"></a>
+                        </div>
+                    </div>
+            </div>
+        </div>
+    </div>
```
Comment (Ibizugbe) [inline index.html]: - [ ] Nice work implementing your modal👍👍 Bu don't forget it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) that the content of your modal be generated dynamically😉😉 kindly go through the[ requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) and make the necessary changes🤗🤗

pop__instruction

- [ ] Also, kindly avoid using inline styling as this is not a good CSS practice😉 kindly take all styles to an external stylesheet🤗
- [ ] I think your popup also looks a bit different from the given design👍👍 kindly confirm with the image below and make the necessary comparison🤗🤗

your popup(mobile)
your_popup mobile

design
popup_desktop

kindly take note of the spacings also😉😉

Same for your desktop version😉😉 take note of the spacings between each contents😉😉😉

File: index.html [hunk 2]
```diff
@@ -126,7 +126,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="project-button pro-btn-one">
-                    <button>See Project</button>
+                    <button class="modal-mobile-see-pro">See Project</button>
```
Comment (Ibizugbe) [inline index.html]: You have done a good job on this milestone👍👍 Just a quick reminder that it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) in this milestone that our project section is generated dynamically using the array of objects you have created in your script file😉😉😉 kindly go through the requirements and make the required change🤗🤗

pop__instruction

File: script.js [hunk 3]
```diff
@@ -6,6 +6,75 @@ const mainBarContent = document.querySelector('.main-bar');
...
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_image3.png',
+  },
+  3: {
+    title: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_image4.png',
+  },
+};
```
Comment (Ibizugbe) [inline script.js]: Nice work implementing the objects👍👍 but I suggest this should be an array of objects so you can easily loop through it and render it dynamically😉😉 also kindly add the `technologies used` and `link to live version` for each objects as these are also part of the contents of the popup and project section😉😉😉

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @alphym 👋👋, 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f69656d6d596a3332456945655a59766a70712f67697068792e676966


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights✅

- [x] Github flow has been used👍
- [x] Linters are checked👍



## Required Changes ♻️

- [ ] Kindly stick to the given design while implementing your popup as this is a requirement for this milestone😉😉

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @alphym , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] [REQUIRED] You have not fixed all the issues raised by the previous reviewer. I encourage you to take a close look at this issue and find a solution for them. 
   - Dynamically display all projects from an array.
   - Dynamically display a modal popup of the correct project button clicked.
   - othe issues raised. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -472,6 +249,61 @@ <h3>Contact me</h3>
...
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                
+                <div class="modal-btn-view">
+                    <a class="modal-mobile-btn see-live">See live <img src="./images/modal-live.png" alt="" style="height: 18px; width: 18px;"> </a>
+                    <a class="modal-mobile-btn see-source"> See source<img src="./images/modal-source.png" alt="" style="height: 18px; width: 18px;"></a>
+                </div>
+            </div>
```
Comment (Favourezeugwa) [inline index.html]: - As suggested by the previous reviewer and part of the project's requirement, you need to implement both your modal and project cards in javaScript. No content of the project card or modal should be in your html file. 🌸 
- You can move this entire modal code to your script.js file

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @alphym  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @alphym, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of GitHub flow ✔️

## Optional changes

- [ ] [Optional] Great effort working on the changes requested by the previous reviewers, you have done wonderfully well creating your project cards and modal pop-up dynamically, but it would have been great if you make use of the `array of an object` method in storing your data rather that the method you used
E.g

```JS
const project = [
  {
    title: 'Tonic',
    description:
            'A daily selection of privately personalized reads; no accounts or sign-ups required.',
    image: './images/project_1.png',
    liveVersion: 'https://alphym.github.io/portfolio/',
    skills: ['html', 'css', 'js'],
    sourceCode: 'https://github.com/alphym/portfolio',
  },
  {
    title: 'Multi-Post Stories',
    description:
            'A daily selection of privately personalized reads; no accounts or sign-ups required.',
    image: './images/project_image2.png',
    liveVersion: 'https://alphym.github.io/portfolio/',
    sourceCode: 'https://github.com/alphym/portfolio',
  },
]
```

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**